koap-core/src/commonTest/kotlin/DecoderTest.kt

```diff
@@ -203,7 +204,7 @@ class DecoderTest {
                 D1 0E # Option Delta: 27, Option Length: 1
                 AB    # Option Value: 3<<4 | 0x8 | 3
             """,
-            expected = Block1(10, true, 128),
+            expected = Block1(Block.Size.`128`, true, 10),
```

Comment:
I feel block number, more flag, block size reads more naturally than reversing the order.
It matches all (big endian) hex output, and the descriptions in the RFC etc.

"it's the 100:th… with more… kilobyte"!

Reply:
Good point.
Definitely agree on the `number` going first.
Wondering though if it'd make sense to have `more` as the last parameter with a default of `false`?

Reply:
I would lean toward rfc order and no default. Anyone using these options would need to know the details anyway, and probably set the more flags explicitly if block-wise is used. Named arguments can always be used, in any order if wanted.